### PR TITLE
swarm/pss: client fix

### PR DESCRIPTION
**Pull request by nolash**
Client implementation was broken after handshake changes:

- Add missing api call GetAddress()
- Use keyid instead of ecdsa.PublicKey in client
- Move client handshake activation to inside client RunProtocol code
- Simplify client handshake

Also implements some lesser cosmetics:

- Added build tag for exluding ping code
- Added "pss" in build tags
- Add conditional compile bool values
- Simplify Ping Protocol code

### CODE REVIEW

File: swarm/pss/client/client.go [hunk 1]
```diff
@@ -285,26 +289,26 @@ func (self *Client) Stop() error {
...
 		symkeyid, err := rw.handshake(handshakeRetryCount, true, true)
-		rw.symKeyId = &symkeyid
 		if err != nil {
 			return err
 		}
 		self.peerPool[topic][pubkeyid] = rw
 		nid, _ := discover.HexID("0x00")
 		p := p2p.NewPeer(nid, fmt.Sprintf("%v", addr), []p2p.Cap{})
 		go self.protos[topic].Run(p, self.peerPool[topic][pubkeyid])
+		_ = symkeyid
```
Comment (lmars) [inline swarm/pss/client/client.go]: Just assign to `_` above.

File: swarm/pss/client/client.go [hunk 2]
```diff
@@ -285,26 +289,26 @@ func (self *Client) Stop() error {
 // The key must exist in the key store of the pss node
 // before the peer is added. The method will return an error
 // if it is not.
-func (self *Client) AddPssPeer(key *ecdsa.PublicKey, addr []byte, spec *protocols.Spec) error {
-	pubkeyid := common.ToHex(crypto.FromECDSAPub(key))
+//func (self *Client) AddPssPeer(key *ecdsa.PublicKey, addr []byte, spec *protocols.Spec) error {
```
Comment (lmars) [inline swarm/pss/client/client.go]: Remove commented code.

File: swarm/pss/ping.go [hunk 3]
```diff
@@ -52,19 +52,20 @@ var PingProtocol = &protocols.Spec{
 
 var PingTopic = ProtocolTopic(PingProtocol)
 
-func NewPingProtocol(pingC chan bool, handler func(interface{}) error) *p2p.Protocol {
+//func NewPingProtocol(pingC chan bool, handler func(interface{}) error) *p2p.Protocol {
```
Comment (lmars) [inline swarm/pss/ping.go]: Remove commented code.

**Review by lmars — APPROVED** [review]

**Review by nonsense — APPROVED** [review]
lgtm

**Pull request merged**